src/main/java/com/es/phoneshop/listeners/ProductDemodataServletContextListener.java

```diff
@@ -0,0 +1,37 @@
+package com.es.phoneshop.listeners;
+
+import com.es.phoneshop.model.product.dao.ArrayListProductDao;
+import com.es.phoneshop.model.product.dao.ProductDao;
+import com.es.phoneshop.model.product.entities.Product;
+
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+import java.math.BigDecimal;
+import java.util.Currency;
+
+public class ProductDemodataServletContextListener implements ServletContextListener {
+    @Override
+    public void contextInitialized(ServletContextEvent servletContextEvent) {
+        ProductDao dao = ArrayListProductDao.getInstance();
```

Comment:
"It should be possible to enable/disable the listener via config parameter in web.xml."